### ISSUE TITLE
[BUG] with dependenci azureml contrib

### ISSUE THREAD

**Issue by miguelgfierro** [microsoft/nlp#387]
### Description

Getting this error in the nightly test pipeline:
```
Pip subprocess error:
  WARNING: azureml-sdk 1.0.48 does not provide the extra 'contribs'
  ERROR: Could not find a version that satisfies the requirement nimbusml==0.6.5; python_version < "3.7" (from azureml-train-automl==1.0.48->-r /data/home/nlpadmin/myagent/_work/6/s/condaenv.y57oin3r.requirements.txt (line 3)) (from versions: 1.1.0, 1.2.0, 1.3.0, 1.4.0)
ERROR: No matching distribution found for nimbusml==0.6.5; python_version < "3.7" (from azureml-train-automl==1.0.48->-r /data/home/nlpadmin/myagent/_work/6/s/condaenv.y57oin3r.requirements.txt (line 3))
```

**Comment by miguelgfierro** [comment]
FYI @saidbleik @hlums

**Comment by saidbleik** [comment]
@heatherbshapiro is this a version mismatch?

**Comment by omri374** [comment]
I tried installing the fix on PR #390 on a fresh DSVM, but still get a message saying:
```
WARNING: azureml-sdk 1.0.57 does not provide the extra 'contribs'
```
This happens with versions 1.0.48 and 1.0.57

I then also tried replacing `contribs` with `contrib` and then got a different message:

```
ERROR: Could not find a version that satisfies the requirement nimbusml==0.6.5; python_version < "3.7" 
```

**Comment by miguelgfierro** [comment]
I made another PR, please take a look. I do get these errors:

```
ERROR: azureml-automl-core 1.0.57 has requirement azureml-dataprep<1.2.0a,>=1.1.10a, but you'll have azureml-dataprep 1.1.8 which is incompatible.
ERROR: azureml-automl-core 1.0.57 has requirement numpy<=1.16.2,>=1.11.0, but you'll have numpy 1.16.4 which is incompatible.
ERROR: azureml-automl-core 1.0.57 has requirement scipy<=1.1.0,>=1.0.0, but you'll have scipy 1.3.1 which is incompatible.
ERROR: azureml-train-automl 1.0.57 has requirement azureml-dataprep<1.2.0a,>=1.1.10a, but you'll have azureml-dataprep 1.1.8 which is incompatible.
ERROR: azureml-train-automl 1.0.57 has requirement numpy<=1.16.2,>=1.11.0, but you'll have numpy 1.16.4 which is incompatible.
ERROR: azureml-train-automl 1.0.57 has requirement scipy<=1.1.0,>=1.0.0, but you'll have scipy 1.3.1 which is incompatible.
ERROR: papermill 1.1.0 has requirement tqdm>=4.32.2, but you'll have tqdm 4.31.1 which is incompatible.
ERROR: azureml-sdk 1.0.57 has requirement azureml-dataprep<1.2.0a,>=1.1.10a, but you'll have azureml-dataprep 1.1.8 which is incompatible.
```
however, then papermill and azure-train-automl are working:

```
python -c "import papermill"
python -c "import azureml.train.automl"
```
